channel.go

```diff
@@ -217,6 +222,12 @@ func NewChannel(serviceName string, opts *ChannelOptions) (*Channel, error) {
 	}
 	ch.peers = newRootPeerList(ch).newChild()
 
+	if opts.Handler != nil {
+		ch.handler = opts.Handler
```

Comment:
If a handler is replaced, should we disallow `Register` on the root channel (it should probably panic?)